modules/javafx.fxml/src/main/docs/javafx/fxml/doc-files/introduction_to_fxml.html

```diff
@@ -763,14 +765,16 @@ <h4><a id="collections_and_property_handlers">Special handlers for collections a
 &lt;VBox fx:controller="com.foo.MyController"
     xmlns:fx="http://javafx.com/fxml" onParentChange="#handleParentChange"/&gt;
 </pre>
-    
+
 <p>Note that collections and properties do not currently support scripting handlers.</p>
 
 <h2><a id="scripting">Scripting</a></h2>
 <p>
 The <span class="code">&lt;fx:script&gt;</span> tag allows a caller to import scripting code into or embed script within a FXML file. Any JVM scripting language can be used, including JavaScript, Groovy, and Clojure, among others. Script code is often used to define event handlers directly in markup or in an associated source file, since event handlers can often be written more concisely in more loosely-typed scripting languages than they can in a statically-typed language such as Java.</p>
 
-<p>For example, the following markup defines a function called <span class="code">handleButtonAction()</span> that is called by the action handler attached to the <span class="code">Button</span> element:</p>
+<p>Note: to turn off automatic compilation of script code place the processing instruction <span class="code">&lt;?compile false?&gt;</span> before the script element. To turn on compilation of script code again use the processing instruction <span class="code">&lt;?compile true?&gt;</span> (or short: <span class="code">&lt;?compile?&gt;</span>). The compile processing instruction can be used repeatedly to turn compilation of script code off and on.</p>
```

Comment:
I would start by saying that scripts are compiled by default if the script engine implements `javax.script.Compilable`. Maybe add a sentence something like this:

Scripts are compiled by default, when they are first loaded, if the `ScriptEngine` implements the `javax.script.Compilable` interface. If compilation fails, the `FXMLLoader` will fall back to interpreted mode.